src/mqt/yaqs/characterization/memory/backends/sequences/workers.py

```
@@ -0,0 +1,547 @@
+#   timesteps_rows          optional per-sequence durations, each length ``num_interventions+1``
+#   operators_list          optional per-sequence MPOs, length ``num_interventions+1`` per sequence
+#   noise_model             None for deterministic surrogate sequences
+#   mcwf_static_ctx         static MCWF context for the whole sequence
+#   mcwf_static_ctx_list    optional per-evolution-slot context (length ``num_interventions+1``)
+#   e_features_rows         per-sequence Choi rows ``(num_interventions, d_e)`` — required for trace workers
+
+
+# ---------------------------------------------------------------------------
+# Process-tensor schedule — ``num_interventions`` interventions, ``num_interventions+1`` evolutions
+# ---------------------------------------------------------------------------
+def _validate_process_tensor_schedule_inputs(
```

Ruff (0.15.20)

[warning] Too many branches (16 > 12)

(PLR0912)